### DISCUSSION TITLE
Can i manage sessions?

### DISCUSSION THREAD

**Discussion by sergeyshaykhullin** [ory/kratos#1006] [Q&A] [answered]
I checked api spec, but found only two methods for session management

Can i list all sessions for identity? Does identity/admin can revoke session by id?

**Comment by vinckr** [comment]
What about the [Administrative Endpoints](https://www.ory.sh/kratos/docs/reference/api#administrative-endpoints) ?

**Reply by sergeyshaykhullin** [reply]
> I checked api spec, but found only two methods for session management

Can you please show an api to list all sessions for identity?

I found only DELETE /sessions, which is admin api without authorization

**Comment by zepatrik** [comment, marked as answer]
Please see #924 and #655
